skpro/distributions/exponential.py

```diff
@@ -0,0 +1,76 @@
+# copyright: skpro developers, BSD-3-Clause License (see LICENSE file)
+"""Exponential probability distribution."""
+
+__author__ = ["ShreeshaM07"]
+
+import pandas as pd
+from scipy.stats import expon, rv_continuous
+
+from skpro.distributions.adapters.scipy import _ScipyAdapter
+
+
+class Exponential(_ScipyAdapter):
+    r"""Exponential Distribution.
+
+    The Laplace distribution is parametrized by mean :math:`\mu` and
+    scale :math:`b`, such that the pdf is
+
+    .. math:: f(x) = \exp\left(-\frac{x - \mu}{b}\right)
+
+    The mean :math:`\mu` is represented by the parameter ``mu``,
+    and the scale :math:`b` by the parameter ``scale``.
+
+    Parameter
+    ---------
+    mu : float or array of float (1D or 2D)
```

Comment:
I would give this a single parameter, `rate`. The `scipy` parameter `scale` is then the inverse of `rate`.

Reply:
I think the exponential distribution takes 2 parameters `loc` and `scale`. If I am not wrong how would having only 1 parameter `rate` work as the distribution is technically `e^-((x-loc)/scale)`.

Reply:
I would suggest, let's use the same parameters as on wikipedia:

https://en.wikipedia.org/wiki/Exponential_distribution

We cannot use `lambda` as it is a reserved python keyword, so `rate` is the next best thing as it is consistent with Poisson process terminology.